### ISSUE TITLE
TS Server fatal error:  Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by stephaneQ2** [microsoft/vscode#116342]
Issue Type: <b>Bug</b>

❗️❗️❗️ Please fill in the sections below to help us diagnose the issue ❗️❗️❗️

**TypeScript Version:** 4.0.6

**Steps to reproduce crash**
1.
2.
3.

**TS Server Log**
❗️ Please review and upload this log file to help us diagnose this crash:
`/Users/quadrant2/Library/Application Support/Code - Insiders/logs/20210210T095901/exthost4/vscode.typescript-language-features/tsserver-log-KEMDtN/tsserver.log`
The log file may contain personal data, including full paths and source code from your workspace. You can scrub the log file to remove paths or other personal information.

**TS Server Error Stack**
Server: `semantic`
```
TypeError: Cannot read property 'indexOf' of undefined
    at Object.parsePackageName (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:38534:30)
    at ConfiguredProject.Project.enablePlugin (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:148757:24)
    at ConfiguredProject.enablePluginsWithOptions (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:149254:30)
    at ProjectService.loadConfiguredProject (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:150963:25)
    at ProjectService.createAndLoadConfiguredProject (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:150902:22)
    at ProjectService.createLoadAndUpdateConfiguredProject (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:150907:36)
    at ProjectService.assignProjectToOpenedScriptInfo (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:151806:44)
    at /Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:152129:97
    at Object.flatMap (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:513:25)
    at ProjectService.applyChangesInOpenFiles (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:152129:41)
    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:153148:46)
    at /Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:155021:88
    at IOSession.Session.executeWithRequestId (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:155012:28)
    at IOSession.Session.executeCommand (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:155021:33)
    at IOSession.Session.onMessage (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:155045:35)
    at Interface.<anonymous> (/Users/quadrant2/Documents/GitHubRepos/KeywordRecognizer/node_modules/typescript/lib/tsserver.js:157261:27)
    at Interface.emit (events.js:315:20)
    at Interface._onLine (readline.js:329:10)
    at Interface._normalWrite (readline.js:474:12)
    at Socket.ondata (readline.js:186:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)
```

VS Code version: Code - Insiders 1.54.0-insider (cd97629c4458c05b16a1fd444d21c61295ba2529, 2021-02-09T08:36:56.595Z)
OS version: Darwin arm64 20.3.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|undefined|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|9, 13, 10|
|Memory (System)|16.00GB (1.36GB free)|
|Process Argv|--crash-reporter-id 95020007-29cd-4ac3-befc-11d5a6e49c42|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (71)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|12.1.4
vscode-angular2-files|ale|1.6.4
ng-template|Ang|11.1.3
vscode-color|ans|0.4.5
azds|azu|2.0.220201212
npm-intellisense|chr|1.3.1
path-intellisense|chr|2.3.0
highlight-line-vscode|cli|0.0.1
bracket-pair-colorizer|Coe|1.0.61
hex-to-rgba|dak|1.0.0
vscode-eslint|dba|2.1.14
githistory|don|0.6.14
xml|Dot|2.5.1
gitlens|eam|11.2.1
vscode-npm-script|eg2|0.3.15
prettier-vscode|esb|5.9.1
todo-tree|Gru|0.0.196
output-colorizer|IBM|0.1.2
angular2-switcher|inf|0.3.3
vscode-nuget-package-manager|jmr|1.1.6
vscode-peacock|joh|3.9.1
firebase-explorer|jsa|0.3.3
vscode-gutter-preview|kis|0.26.2
vscode-dashboard|kru|2.3.1
regionfolder|map|1.0.12
Angular-BeastCode|Mik|10.0.3
file-downloader|min|1.0.9
mindaro|min|1.0.120210126
vscode-autoprefixer|mrm|3.0.1
vscode-azureappservice|ms-|0.20.0
vscode-azurefunctions|ms-|1.2.1
csharp|ms-|1.23.9
vscode-aks-tools|ms-|0.0.8
vscode-kubernetes-tools|ms-|1.2.4
remote-containers|ms-|0.158.0
remote-ssh|ms-|0.63.0
remote-ssh-edit|ms-|0.63.0
remote-wsl|ms-|0.53.4
vscode-remote-extensionpack|ms-|0.20.0
azure-account|ms-|0.9.5
cpptools|ms-|1.2.0
vs-keybindings|ms-|0.2.0
vscode-typescript-tslint-plugin|ms-|1.3.3
vsliveshare|ms-|1.0.3629
vsliveshare-audio|ms-|0.1.91
vsliveshare-pack|ms-|0.4.0
debugger-for-chrome|msj|4.12.12
debugger-for-edge|msj|1.0.15
angular2-inline|nat|0.0.17
nativescript|Tel|0.11.0
quicktype|qui|12.0.46
vscode-data-preview|Ran|2.2.0
java|red|0.74.0
vscode-yaml|red|0.15.0
angular-follow-selector|san|1.2.0
code-settings-sync|Sha|3.4.3
autoimport|ste|1.5.3
code-spell-checker|str|1.10.2
move-ts|str|1.12.0
tabnine-vscode|Tab|3.2.7
highlight-matching-tag|vin|0.10.0
vscodeintellicode|Vis|1.2.11
vscode-java-debug|vsc|0.31.0
vscode-java-dependency|vsc|0.18.0
vscode-java-pack|vsc|0.12.1
vscode-java-test|vsc|0.28.0
vscode-maven|vsc|0.27.1
vscode-icons|vsc|11.1.0
vscode-wakatime|Wak|5.0.0
change-case|wma|1.0.0
html-css-class-completion|Zig|1.20.0

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30208930
vstry914:30244316
pythonvsdeb440:30224570
pythonvsded773:30223139
pythonvspyt875:30251590
core-portspanelcf:30233468
coreuntitledfile:30249963

```

</details>

**Comment by archiekd** [comment]
@stephaneQ2 

I found a temp fix here: https://github.com/microsoft/vscode/issues/116219#issuecomment-776407990 

More info in this issue: #116219

**Comment by gjsjohnmurray** [comment]
/duplicate of #116219 and will be fixed in upcoming 1.53.2
